src/kinesis/consumer.py

```diff
@@ -198,6 +201,7 @@ def __iter__(self):
                             if not self.run:
                                 break
 
+                            item['ShardId']=shard_id
                             log.debug(item)
                             yield item
```

Comment:
I do not like the idea mutating the item.

Instead of adding the `ShardId` to the item what do you think about adding another option to init (something like `return_shard_id=False`) that if true would change the yield to:

```
if self.return_shard_id:
    yield shard_id, item
else:
    yield item
```